### DISCUSSION TITLE
Media Player Group Populated with my Garage Door.

### DISCUSSION THREAD

**Discussion by ImaBadCoder** [jseidl/magic-areas#291] [Q&A]
I have enabled media player groups, the only entity that is added is my garage door cover.. cover.garage_door_controller.. 

Any help understanding this logic or helping troubleshoot is appreciated.

Larry

**Comment by ImaBadCoder** [comment]
Maybe I will submit this as a bug.

**Comment by jseidl** [comment]
Hi, if this is still happening please open a bug on the issue tracker
